README.md

```diff
@@ -49,7 +49,14 @@ surfnet_saml:
             certificate_file: "%saml_remote_idp_certificate%" 
 ```
 
-See [Saml bundle documentation](https://github.com/OpenConext/Stepup-saml-bundle) for more information about this configuration
+See [Saml bundle documentation](https://github.com/OpenConext/Stepup-saml-bundle) for more information about the configuration above
+
+```yaml
+assetic:
```

Comment:
I thought you would have chosen the more modern [Encore](https://symfony.com/blog/introducing-webpack-encore-for-asset-management). (webpack based) frontend asset management solution. Im totaly fine with the Assetic solution, but you might want to reconsider.